_includes/tutorial-content.html

```diff
@@ -5,36 +5,31 @@
         <div class="column is-full">
           <h2 class="title tutorial-section-title" id="run-it">Run it</h2>
 
-          {% for step in site.data.harnesses[page.static_data][page.stack].dev.steps %}
+          {% for step in
+          site.data.harnesses[page.static_data][page.stack].dev.steps %}
```

Comment:
nit: Seems like your IDE reformat this. Personally, prefer the one from before.

Reply:
cc: @scott-confluent is working on this in another PR